### PR TITLE
CMS-304 template changes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-304

Before:
screen shot 2015-09-16 at 14 23 56
screen shot 2015-09-16 at 14 24 00
After:
screen shot 2015-09-16 at 14 21 35

Before:
screen shot 2015-09-16 at 14 24 13

After:
screen shot 2015-09-16 at 14 22 10

### CODE REVIEW

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 1]
```diff
@@ -52,18 +52,40 @@
...
-<div id="content-main">
+<div id="content-top-bar">
+    {% if not folder.is_root %}
+            <h1 class="folder_header">
+                {% if folder.parent %}
+                    <a class="backtoroot" href="{% url 'admin:filer-directory_listing' folder.parent.id %}{% if is_popup %}?_popup=1{% if select_folder %}&select_folder=1{% endif %}{% endif %}" title="{% trans "Go back to the parent folder" %}">&uarr;</a>
+                {% else %}
+                    <a class="backtoroot" href="{% url 'admin:filer-directory_listing-root' %}{% if is_popup %}?_popup=1{% if select_folder %}&select_folder=1{% endif %}{% endif %}" title="{% trans "Go back to" %} {% trans "root"|title %} {% trans "folder" %}">&uarr;</a>
+                {% endif %}
+                <img src="{{ folder.icons.32 }}" alt="{% trans "Folder Icon" %}" /> {{ folder.name }}
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_listing.html]: do we really want the "alt" Folder Icon?

File: filer/templates/admin/filer/folder/directory_table.html [hunk 2]
```diff
@@ -11,6 +11,9 @@
                 <th class="thumbHeader">&nbsp;</th>
                 <!-- FILENAME / FOLDERNAME / DIMENSIONS / OWNER -->
                 <th>{% trans 'Name' %}</th>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: double quotes?

File: filer/templates/admin/filer/folder/directory_table.html [hunk 3]
```diff
@@ -23,11 +26,18 @@
                 <td><a href="{{ subfolder.get_admin_directory_listing_url_path }}{% if is_popup %}?_popup=1{% if select_folder %}&select_folder=1{% endif %}{% endif %}" title="{% blocktrans with subfolder.name as item_label %}Change '{{ item_label }}' folder details{% endblocktrans %}"><img src="{{ subfolder.icons.48 }}" alt="{% trans "Folder Icon" %}" /></a></td>
                 <!-- Directory details -->
                 <td>
+                    <div><b><a href="{{ subfolder.get_admin_directory_listing_url_path }}{% if is_popup %}?_popup=1{% if select_folder %}&select_folder=1{% endif %}{% endif %}" title="{% blocktrans with subfolder.name as item_label %}Change '{{ item_label }}' folder details{% endblocktrans %}">{{ subfolder.name }}</a></b></div>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: not `<b>` rather `<strong>`

**Comment by FinalAngel** [comment]
LGTM

**Comment by yakky** [comment]
@lory87 LGTM

**Comment by mkoistinen** [comment]
Look real good!

**Pull request merged**